### ISSUE TITLE
3.0.1 was not published to PyPI

### ISSUE THREAD

**Issue by asilvestroni** [MrBin99/django-vite#107]
Hi, it seems like the PyPI publish job for 3.0.1 failed https://github.com/MrBin99/django-vite/actions/runs/6989145161.

Would it be possible to publish the new version in some other way? 
By the way really dig the project! Hope I can have it running again soon since my Vite manifest has changed and is not supported anymore.

**Comment by thijskramer** [comment]
Hi @asilvestroni, 

Thanks for your issue! I didn't notice... 

Unfortunately the upload fails with the following error:
```
ERROR    HTTPError: 403 Forbidden from https://upload.pypi.org/legacy/          
         Invalid or non-existent authentication information. See                
         https://pypi.org/help/#invalid-auth for more information.             
```

@MrBin99 Do you know what's going on? And are you able to fix it?

**Comment by MrBin99** [comment]
Arg,
It seems like Pypi deprecated authentication (with Twine) with username/password.
I will replace with the official workflow for publishing.

**Comment by MrBin99** [comment]
This is fixed, the 3.0.1 is published.
Thanks for the report.
